### DISCUSSION TITLE
ItemsAdder Support

### DISCUSSION THREAD

**Discussion by dinobossytnew** [Lorenzo0111/QualityArmory#655] [Q&A] [answered]
### Confirmations

- [X] There isn't any other PR or issue regarding this.
- [X] This feature does not exist in the latest version.

### What is your request about?

Other

### Description

To make it easier to integrate 3d models I use itemsadder but the customdatamodel is not always seamless with ItemsAdder so I would like you to add a new configuration option called:

ItemsAdder:namespace/nameitemmodel

so it would take the item itself with the selected model without any slowness and quite fast.

**Comment by JaHollTV** [comment, marked as answer]
@dinobossytnew 
I don't know if this feature is really still needed:

We have been using ItemsAdder and QA successfully together for years.
We insert the weapon models in ItemsAdder as a crossbow and then set the ID in the respective QA config of the weapon to the ModelData that ItemsAdder has assigned.

Only the QA weapon can then shoot and not the item you can give yourself via IA, but that is perfectly sufficient, at least for our purposes.
